src/solverpip.ts

```diff
@@ -32,7 +44,8 @@ function satisfies(version: string, constraint: string) {
       return false;
     }
 
-    const [, operator, constraintVersion] = match;
+    let [, operator, constraintVersion] = match;
+    constraintVersion = formatConstraintVersion(constraintVersion, version);
     const cmp = compareVersions(version, constraintVersion);
```

Comment:
Nitpicking formatting

```suggestion
    const [, operator, constraintVersion] = match;
    const cmp = compareVersions(version, formatConstraintVersion(constraintVersion, version));
```